README.md

```diff
@@ -79,11 +72,37 @@ The MoveIt! Motion Planning Framework **for ROS 2.0**
   - [ ] Convert moveit_task_constructor
   - [ ] Convert moveit_resources
   - [ ] Convert moveit_commander
-  - [ ] Convert moveit_fake_controller_manager
+  - [ ] Convert moveit_fake_controller_manager  
 - [ ] New features in ROS 2.0
   - [ ] Migrate plugin architecture to ROS2 nodelets
 - [ ] Documentation
-  - [ ] Create tutorials
+  - [ ] Update tutorials for MoveIt2
+  - [ ] Create tutorial on using ros1/ros2 bridge to support ros1 hardware drivers
+  - [ ] Move install instructions to moveit.ros.org
+- [ ] Major refactoring and divergence from moveit2
```

Comment:
@rhaschke some of the breaking changes i'm hoping to do for moveit2, to be discussed

Reply:
Apart from moving to ROS2 structures instead of plugin interfaces - and this is not even listed here - I do not see a single item in this list I would hesitate to discuss for moveit-for-ros1's `master` branch too.

Reply:
ok, well let's get past the ros1 release and master branch split and discuss that there!